### PR TITLE
feat(@vtmn/css): add snackbar component

### PR DESCRIPTION
**Pull request by GaspardMathon**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] Check PR name, it must follow the https://www.conventionalcommits.org pattern
- [x] If it's a new component in CSS, ask for design review

## Description

- Add snackbar component

## Does this introduce a breaking change?

- No

❤️ Thank you!

### CODE REVIEW

**Comment by xavhan** [comment]
I removed the frontend-community review as PR is not ready yet

File: packages/sources/css/src/components/snackbar/src/index.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  padding: rem(8px) rem(16px);
+  height: rem(48px);
+  max-width: fit-content;
+  font-family: var(--vtmn-typo_font-family);
+  font-size: var(--vtmn-typo_text-2-font-size);
+  line-height: var(--vtmn-typo_text-2-line-height);
+  color: var(--vtmn-semantic-color_content-primary-reversed);
+  background-color: var(--vtmn-semantic-color_background-primary-reversed);
+  box-shadow: var(--vtmn-shadow_200);
+  border-radius: rem(4px);
```
Comment (xavhan) [inline packages/sources/css/src/components/snackbar/src/index.css:20]: border radius is dependent of the root font size ? should it be constant and defined in px instead ?

Comment (GaspardMathon) [reply packages/sources/css/src/components/snackbar/src/index.css:22]: Thanks for feedback Xavier. I created an issue to rework the pixels sizes.

File: packages/sources/css/src/components/snackbar/src/index.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.vtmn-snackbar > .vtmn-btn {
+  margin-left: rem(16px);
+}
+
+.vtmn-snackbar_content {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
```
Comment (xavhan) [inline packages/sources/css/src/components/snackbar/src/index.css:41]: dont you have a dedicated tailwind helper for that ? 
vtmn-text-truncate or something ? https://tailwindcss.com/docs/text-overflow#truncate

File: packages/sources/css/src/design-tokens/src/animations.css [hunk 3]
```diff
@@ -4,13 +4,44 @@
  */
 :root {
   --vtmn-animation_fade-in: fade_in 200ms ease-in-out forwards;
```
Comment (lauthieb) [inline packages/sources/css/src/design-tokens/src/animations.css:6]:
```suggestion
  --vtmn-animation_fade-in: fade-in 200ms ease-in-out forwards;
```

**Review by lauthieb — CHANGES_REQUESTED** [review]

**Review by lauthieb — APPROVED** [review]

**Comment by lauthieb** [comment]
@GaspardMathon great job! :tada:

**Pull request merged**